scrapy/downloadermiddlewares/httpcompression.py

```diff
@@ -73,7 +76,10 @@ def _decode(self, body, encoding):
                 # http://www.gzip.org/zlib/zlib_faq.html#faq38
                 body = zlib.decompress(body, -15)
         if encoding == b'br' and b'br' in ACCEPTED_ENCODINGS:
-            body = brotli.decompress(body)
+            if (flag):
+                body = brotli.decompress(body)
+            else:
+                print("brotli not installed")
```

Comment:
It should be a warning here, not a simple `print`.
To some extent, it could even be an error as it will fail when processing the response, as it wasn't decoded properly.